module4/assignment4.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Assignment #4</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<link rel="stylesheet" href="_css/assignment4theme.css"/>
```

Comment:
custom theme! However - it doesn't look like it's applying when I view your app on the web server...

Reply:
> custom theme! However - it doesn't look like it's applying when I view your app on the web server...
I'm not sure why that might be?  Do you, because it would be helpful for this applied challenge